data_processors/pipeline/lambdas/demux.py

```diff
@@ -0,0 +1,92 @@
+try:
+    import unzip_requirements
+except ImportError:
+    pass
+
+import django
+import os
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings.base')
+django.setup()
+
+# ---
+
+import logging
+
+from data_processors.pipeline.constant import FastQReadType, WorkflowType
+from data_processors.pipeline.lambdas import fastq, germline
+from utils import libjson
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+SUPPORTED_DEMUX_WORKFLOWS = [
```

Comment:
Minor comment: demux = demultiplexing = FASTQ generation from raw sequencing data (BCL/cBCL files) = BCL convert/bcl2fastq
(the other workflows, like germline, T/N, etc, that start with FASTQ files as input are usually analysis workflows)

Reply:
Ah. Right, ok.